src/DOM/HTML/Window.purs

```diff
@@ -13,3 +42,71 @@ foreign import location :: forall eff. Window -> Eff (dom :: DOM | eff) Location
 foreign import innerWidth :: forall eff. Window -> Eff (dom :: DOM | eff) Int
 
 foreign import innerHeight :: forall eff. Window -> Eff (dom :: DOM | eff) Int
+
+foreign import alert :: forall eff. Window -> String -> Eff (alert :: ALERT | eff) Unit
+
+foreign import close :: forall eff. Window -> Eff (window :: WINDOW | eff) Unit
+
+foreign import closed :: forall eff. Window -> Eff (window :: WINDOW | eff) Boolean
```

Comment:
Please remove this, it's a nonstandard property - this library is intentionally only implementing standards, even when things are commonly available.